diesel_codegen/Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository = "https://github.com/diesel-rs/diesel/tree/master/diesel_codegen"
 keywords = ["orm", "database", "postgres", "sql", "codegen"]
 
 [dependencies]
-diesel_codegen_syntex = { version = "0.7.0", default-features = false }
+diesel_codegen_syntex = { path = "../diesel_codegen_syntex", version = "0.7.3", default-features = false }
```

Comment:
It's not a breaking change in the semver sense, I believe. I bumped the version since 0.7.3 will work with the latest nightly while 0.7.2 will not.